org.freedesktop.Sdk.Extension.deno.yaml

```diff
@@ -0,0 +1,61 @@
+id: org.freedesktop.Sdk.Extension.deno
+branch: '23.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+ prefix: /usr/lib/sdk/deno
+ env:
+    PATH: /usr/lib/sdk/deno/bin:/app/bin:/usr/bin
+modules:
```

Comment:
You did not install the appdata

Reply:
I added `      - install -Dm644 ${FLATPAK_ID}.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo` is that correct ?